packages/mongodb-redact/src/regexes.ts

```diff
@@ -38,18 +38,16 @@ var regexes = [
 
   // URLs
   [
-    /(http(s)?:\/\/)(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}(\.[a-z]{2,6})?\b([-a-zA-Z0-9@:%_\+.~#?&\/\/=]*)/gim,
+    /(http(s)?:\/\/)(www\.)?[-a-zA-Z0-9@:%._+~#=]{2,256}(\.[a-z]{2,6})?\b([-a-zA-Z0-9@:%_+.~#?&/=]*)/gim,
     '<url>',
   ],
 
   // MongoDB connection strings
   [
-    /(mongodb:\/\/)(www\.)?[-a-zA-Z0-9@:%._\+~#=,]{2,256}(\.[a-z]{2,6})?\b([-a-zA-Z0-9@:%_\+.~#?&\/\/=]*)/gim,
+    /(mongodb:\/\/)(www\.)?[-a-zA-Z0-9@:%._+~#=,]{2,256}(\.[a-z]{2,6})?\b([-a-zA-Z0-9@:%_+.~#?&/=]*)/gim,
```

Comment:
The end part here `[-a-zA-Z0-9@:%_+.~#?&/=]` used to contain a `\/` character`[-a-zA-Z0-9@:%_\+.~#?&\/\/=]` . Is that not needed anymore?

Reply:
These are all automated replacements, no manual changes – `\/` inside a `[]` is equivalent to just `/` and `//` inside `[]` is equivalent to `/`.

Reply:
(This did make me wonder if there was an intention to target a different character, since listing `/` twice seems odd. But I wanted to avoid functional changes in this PR, it's just for moving things to monorepo/TypeScript)